js/jquery.validation.js

```diff
@@ -227,7 +230,7 @@
             var $scrollElement = undefined;
             // Check each record item is valid.
             $.each(response.record, function (name, object) {
-                var $element = $form.find('[name=' + name + ']');
+                var $element = $form.find('[name=' + name + ']:not(input[type=hidden])');
                 if ( ! object.valid ) {
```

Comment:
hidden inputs are added for submit buttons so that form variables are included in the request.

Validation qtips should not be shown against the hidden inputs that have the same name attribute